content/overview/workflow/process.md

```diff
@@ -0,0 +1,209 @@
+
+### dodge and burn
+
+Dodging and burning is a traditional darkroom technique to add and remove brightness from an image. There are two recommended ways to achieve this
+
+- If you want to selectively dodge or burn only certain objects you can apply a [new instance](../../darkroom/processing-modules/multiple-instances.md) of the [_exposure_](../../module-reference/processing-modules/exposure.md) module using a [drawn mask](../../darkroom/masking-and-blending/masks/drawn.md) to isolate the effect to the required area of the image (see also the [mask refinement](../../darkroom/masking-and-blending/masks/refinement-controls.md) section for more information). Move the exposure slider to alter the brightness of the masked area.
+- If you wish to dodge or burn areas with a similar brightness (for example, to brighten the shadows or darken the highlights) you should use the [_tone equalizer_](../../module-reference/processing-modules/tone-equalizer.md) module.
+
+### convert to monochrome
+
+darktable provides a number of ways to remove the color from your image. The most flexible method is to use the [_gray tab of the color calibration module_](../../module-reference/processing-modules/color-calibration.md#gray-tab). A number of film emulation presets are available in this module to provide you with a starting point.
+
+See the [developing monochrome images](../../guides-tutorials/monochrome.md) section for details of other techniques.
+
+### color grading
+
+The [_color balance rgb_](../../module-reference/processing-modules/color-balance-rgb.md) module is your one-stop-shop for controlling the colors in your image. Adjustments can be isolated to the shadows, highlights and mid-tones, or applied at a global level.
+
+# other important topics
+
+## re-use common module settings
+
+If you find yourself using the same module parameters repeatedly, you can create [presets](../../darkroom/processing-modules/presets.md) containing your favorite settings. If you use the same settings on every image, you can also make presets apply automatically to new images.
+
+You may also have groups of module settings that you commonly apply only to certain types of image. You can use [styles](../../module-reference/utility-modules/lighttable/styles.md) to apply multiple module settings at once to a selection of images.
+
+## perform adjustments locally
+
+Most darktable modules can either be applied to the whole image, or restricted to parts of the image using [drawn and parametric masks](../../darkroom/masking-and-blending/masks/_index.md).
+
+## control darktable with other input methods
+
+You don't need to use the darktable UI to make adjustments to your images. Most of the functionality in darktable can also be controlled using shortcuts defined with your keyboard/mouse and even with other input devices such as game controllers and MIDI devices. See the [shortcuts](../../preferences-settings/shortcuts.md) section for details.
```

Comment:
again, a bit more detail might be useful here. eg. talk about the value of using a mouse with a scroll wheel as opposed to a touchpad. maybe an example or two may help the user better understand the usefulness of this.

Reply:
I'd rather keep whys and wherefores out of the introductory section. This is more about flagging an important reference section to read.